.github/workflows/test.yml

```diff
@@ -0,0 +1,108 @@
+name: docs-runme
```

Comment:
We should call the file like the run, so either call this `test` or the file `docs-runme.yaml`, I would suggest calling this `test-examples` or `e2e-examples`

Reply:
updated 🫡